diff --git a/coffee-mode.el b/coffee-mode.el
index b819d75..2ec0ec6 100644
--- a/coffee-mode.el
+++ b/coffee-mode.el
@@ -479,7 +479,7 @@ called `coffee-compiled-buffer-name'."
     (,coffee-prototype-regexp . font-lock-variable-name-face)
     (,coffee-assign-regexp . font-lock-type-face)
     (,coffee-local-assign-regexp 1 font-lock-variable-name-face)
-    (,coffee-regexp-regexp . font-lock-constant-face)
+    ;(,coffee-regexp-regexp . font-lock-constant-face)
     (,coffee-boolean-regexp . font-lock-constant-face)
     (,coffee-class-regexp 2 'coffee-mode-class-name)
     (,coffee-keywords-regexp 1 font-lock-keyword-face)))
